test/meson.build

```diff
@@ -0,0 +1,23 @@
+# Rules for building tests
```

Comment:
I guess, a `.travis.yml` would be good to have too, so that TravisCI could run the tests automatically on each commit.

Reply:
Added github workflow